e2e/tests/core/04-channel/upgrades_test.go

```diff
@@ -130,6 +188,17 @@ func (s *ChannelTestSuite) createUpgradeFields(channel channeltypes.Channel) cha
 	return channeltypes.NewUpgradeFields(channel.Ordering, channel.ConnectionHops, string(versionBytes))
 }
 
+// setUpgradeTimeoutParam creates and submits a governance proposal to execute the message to update 04-channel params with a timeout of 1s
+func (s *ChannelTestSuite) setUpgradeTimeoutParam(ctx context.Context, chain ibc.Chain, wallet ibc.Wallet) {
+	govModuleAddress, err := s.QueryModuleAccountAddress(ctx, govtypes.ModuleName, chain)
+	s.Require().NoError(err)
+	s.Require().NotNil(govModuleAddress)
+
+	upgradeTimeout := channeltypes.NewTimeout(channeltypes.DefaultTimeout.Height, 1000000000)
```

Comment:
This is what triggers the timeout right? because this is this 1 second? Should we add an inline comment?

```suggestion
	upgradeTimeout := channeltypes.NewTimeout(channeltypes.DefaultTimeout.Height, 1000000000) // set to 1 second to trigger a timeout
```

Reply:
could also set it as a top level const!

Reply:
This setting this to 1 second will trigger the timeout. I will address both comments. Thanks!